### ISSUE TITLE
master won't compile

### ISSUE THREAD

**Issue by jpdna** [bigdatagenomics/adam#1200]
As of d70e68b  from earlier today 
ADAM master won't compile for me

```
[ERROR] /home/jp/test_compile/v1/adam/adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala:297: error: not found: value resourcePath
[ERROR]     val inputPath = resourcePath("HLA_DQB1_05_01_01_02.fa")
[ERROR]                     ^
[ERROR] one error found
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.10 .......................................... SUCCESS [  4.943 s]
[INFO] ADAM_2.10: Core .................................... FAILURE [ 37.906 s]
[INFO] ADAM_2.10: APIs for Java ........................... SKIPPED
[INFO] ADAM_2.10: CLI ..................................... SKIPPED
[INFO] ADAM_2.10: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

```

As a control, the earlier commit b903cfd
does compile fine for me

**Comment by fnothaft** [comment]
Oh drat! Thanks for catching this @jpdna and thanks for the quick fix @ryan-williams. We merged a PR without retesting it after a previous PR merged. Good to be aware of this error model...

**Comment by ryan-williams** [comment]
Funny, #1198 added a new call of a method (`resourceFile`) that was removed by #1142.

We didn't notice because GitHub/git just do a relatively simple lexical pass to determine whether two commits conflict, and this is a pretty reliable way to get that algorithm to generate false-negatives for conflicts, resulting in build breaks like this.

I sent the (trivial) fix in #1201.

<rant> The longer term fix is to live in a world where programming tools exist that interpret commits at the AST level or higher, so that "remove `resourcePath` and reroute all calls to `testFile`" would be automatically inferred as conflicting with a commit that included new calls to `resourcePath` (or automatically resolved correctly), but alas, that's probably a few years out! </rant>

**Comment by fnothaft** [comment]
> The longer term fix is to live in a world where programming tools exist that interpret commits at the AST level or higher, so that "remove resourcePath and reroute all calls to testFile" would be automatically inferred as conflicting with a commit that included new calls to resourcePath (or automatically resolved correctly), but alas, that's probably a few years out!

That would be pretty fantastic!
